src/redux/actions/instructorActions.js

```diff
@@ -35,7 +35,8 @@ export const getusers = () => async dispatch => {
 };
 export const getCourses = () => async dispatch => {
   try {
-    const res = await axiosWithAuth().get(`/class-instances`);
+    const res = await axiosWithAuth().get(`/courses`);
```

Comment:
I might be mistaken, but I believe this won't work with the newly fixed axiosWithAuth since it requires an idToken as an argument. Should definitely give this a double check.